client/src/components/Calendar.js

```diff
@@ -16,66 +16,81 @@ const Form = styled.div`
 `;
 
 const DirectionButton = styled.button`
-  background-color: #011e62;
+  background-color: ${colors.primaryColor};
   border: none;
   outline: none;
   height: 20px;
-  margin-left: 5px;
-  margin-right: 5px;
+  margin: 0px 5px 0px 5px;
 `;
 
 const DayText = styled.p`
-  font-family: Roboto;
-  font-size: 15px;
-  color: white;
+  font-size: 0.937rem;
```

Comment:
You can style the text inside a button from the styled component button. just include these props there and it should still work